app/Website/src/Http/Requests/ContactFormRequest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Website\Http\Requests;
+
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class ContactFormRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        $rules = [
+            'first_name' => 'required|string|max:255',
+            'last_name' => 'required|string|max:255',
+            'email' => 'required|email:filter',
+            'subject' => 'required|string|max:255',
+            'message' => 'required|string|max:1000',
+        ];
+
+        if (app()->environment('production')) {
```

Comment:
You can use RequiredIf validation rule instead of using this if statement here. Check the docs for more info.

Reply:
```
'cf-turnstile-response' => [
                new RequiredIf(function () {
                    return app()->environment('production');
                }),
                Rule::when(app()->environment('production'), [
                    'string',
                    Rule::turnstile(),
                ]),
            ],
```

Is this okay?

Reply:
Yeah, that should be good. You can use `app()->isProduction()` as well.